RELEASES.md

```diff
@@ -1,3 +1,122 @@
+Version 1.37.0 (2019-08-15)
+==========================
+
+Language
+--------
+- [`#[must_use]` will now warn if the type is in a tuple and unused.][61100]
+- [`#[must_use]` will now warn if the type is in a `Box` and unused.][62228]
+- [`#[must_use]` will now warn if the type is in a array and unused.][62235]
+- [You can now use the `cfg` and `cfg_attr` attributes on
+  generic parameters.][61547]
+- [You can now use enum variants through type alias.][61682] e.g. You can
+  write the following.
+  ```rust
+  type MyOption = Option<u8>;
+
+  fn increment_or_zero(x: MyOption) -> u8 {
+      match x {
+          MyOption::Some(y) => y + 1,
+          MyOption::None => 0,
+      }
+  }
+  ```
+- [You can now use `_` as a identifier for consts.][61347] e.g. You can write
+  `const _: u32 = 5;`.
+- [You can now use `#[repr(align)]` on enums.][61229]
+- [The  `?`/_"Kleene"_ macro operator is now available in the
+  2015 edition.][60932]
+
+Compiler
+--------
+- [You can now enable Profile-Guided Optimization with the `-C profile-generate`
+  and `-C profile-use` flags.][61268] For more information on how to use profile
+  guided optimization, please refer to the [rustc book](rustc-book-pgo).
+- [The `rust-lldb` wrapper script should now work again.][61827]
+
+Libraries
+---------
+- [`mem::MaybeUninit` is now `#[repr(transparent)]`.][61802]
```

Comment:
Cc @cramertj, should we mention this?

Reply:
Meh-- seems fine to include or leave out, but maybe if we include it the text could be something like "`MaybeUninit<T>` is now ABI-compatible with `T`."